flow360/examples/tutorial_bet_disk.py

```diff
@@ -0,0 +1,16 @@
+"""
+tutorialBetDisk meshing example
+"""
+
+from .base_test_case import BaseTestCase
+
+
+class TutorialBETDisk(BaseTestCase):
+    name = "tutorialBetDisk"
+
+    class url:
+        geometry = "local://geometry.csm"
+        surface_json = "local://surface_params.json"
+        volume_json = "local://volume_params.json"
+        case_json = "local://case_params.json"
+        extra = {"disk0": "disk0.json", "cylinder0": "cylinder0.json"}
```

Comment:
Same comment as in https://github.com/flexcompute/Flow360/pull/572#discussion_r1856863644

 Regarding the json files, what is the purpose of storing them locally? It seems that we only use the disk0.json file for the BETDisk definition. Can we only keep the essitial ones and remove the files that are not used?

Reply:
In addition, I feel that using the disk0.json and cylinder.json file as the starting point of the BETDisk case may confuse both existing and new users since it is not clear where these files are from. Given both files can be generated using the conversion function, can we directly call the conversion function in the tutorial script to generate these files? In this way, we use the old "Flow360.json" file as the starting point of the BETDisk case and make this script to serve the existing users and help them migrate.

Later when the function to define BETDisk from an input XROTOR (or any other formats) file is ready, we can update this example script to show the desired workflow for the BETDisk.

Reply:
I see, then we keep disk0.json and cylinder0.json as is.